.sequelizerc

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('module-2/config', 'database.json'),
```

Comment:
It's better to have some logic that changes the config json file depending on the environment you're using.
Like "config/dev.json" and "config/prod.json" - obviously, "prod.json" should not be commited to source.
Therefore, consider investing some time to create such logic.

Reply:
Well per the tutorials I used to implement this portion, other database configs could be added to existing `database.json` file under environment name keys like `'production' | 'test' | 'development'` and those env configs will be applied if `NODE_ENV` env variable is passed and `development` will be used by default. This logic is put under `module-2/DAL/index.ts`. Please let me know if there are some downsides to this technique.

Reply:
I see.
This is fine as long as you're OK with all the configs will be in one object.